libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -693,23 +693,35 @@ void AP_GPS::update(void)
                 // handle switch between real GPSs
                 for (uint8_t i=0; i<GPS_MAX_RECEIVERS; i++) {
                     if (i == primary_instance) {
-                        continue;
+                        continue; 
```

Comment:
remove space after semi-colon

Reply:
That's where second pair of eyes comes in ;) 
Thanks